src/ui/components/results/verticalresultscomponent.js

```diff
@@ -243,6 +258,53 @@ export default class VerticalResultsComponent extends Component {
     return 'results/verticalresults';
   }
 
+  /**
+   * Given an array of nlp filters from the backend (see class AppliedQueryFilter within section.js),
+   * turn them into an array of SimpleFilterNodes
+   * @param {Array<Object>} nlpFilters
```

Comment:
Array< AppliedQueryFilter >, then you can get rid of the "(see class ...)"

Reply:
updated!